pkg/storage/doc.go

```diff
@@ -0,0 +1,11 @@
+// Package storage implements the storage layer of the application.
+// It contains the database operations for the application.
+// It interacts with the model layer to perform CRUD operations on the database.
+// It provides an abstraction over the database operations to the service layer.
+//
+// The storage layer should be used by the service layer to perform database operations.
+//
+// The storage layer contains the following packages:
```

Comment:
no need to describe this here, this is a separate package which is `inmemory`. you can have one mode doc.go file where you will describe about in memory db.